docs/ops.html

```diff
@@ -1931,6 +1931,27 @@ <h4 class="anchor-heading"><a id="tiered_storage_monitoring" class="anchor-link"
         <td>Average idle percent of thread pool for copying data to remote storage</td>
         <td>kafka.log.remote:type=RemoteLogManager,name=RemoteLogManagerTasksAvgIdlePercent</td>
       </tr>
+      <tr>
+        <td>RemoteLogManager Avg Broker Throttled Fetch Time</td>
```

Comment:
Should we have this something like - `RemoteLogManager Avg Broker Fetch Throttle Duration`